src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/map/MapControl.java

```diff
@@ -50,15 +50,15 @@
 	 *
 	 */
 	@ElementCollection
-	@MapKeyColumn(name = "MAPCTRLPROPERTY")
+	@MapKeyColumn(name = "PROPERTY")
 	@Column(name = "VALUE")
 	@CollectionTable(
-			name = "MAPCONTROL_MAPCTRLPROPERTIES",
-			joinColumns = @JoinColumn(name = "MAPCONTROL_ID")
+		name = "MAPCONTROLS_PROPERTIES",
```

Comment:
Because the implicitly generated value in this case would exceed 30 chars (but the PhysicalNamingStrategy cuts off after 30 chars in case of oracle)